salt/core/mail.sls

```diff
@@ -0,0 +1,51 @@
+# Install and configure email software
+postfix:
+  pkg:
+    - installed
+  service:
+    - running
+    - require:
+      - pkg: postfix
+  debconf.set:
+    - data:
+        # I don't expect FQDN to ever be unset but if it is, fall back to /etc/mailname
+        'postfix/mailname': {'type': 'string', 'value': {{ salt['grains.get']('fqdn','/etc/mailname') }} }
+        # Internet Site = Mail is sent and received directly using SMTP
+        'postfix/main_mailer_type': { 'type' : 'select', 'value' : 'Internet Site' }
+    - require_in:
+      - pkg: postfix
+    - require:
+      - pkg: debconf-utils
+  cmd.wait:
+    - name: dpkg-reconfigure -f noninteractive postfix
+    - watch:
+      - debconf: postfix
+    - require:
+      - pkg: postfix
+
+# Install commands for users to interact with mail
+mailutils:
+  pkg.installed
+
+# Configure /etc/aliases
+root:
+  alias.present:
+    - target: {{ pillar['system_contacts']['root'] }}
+admin:
+  alias.present:
+    - target: root
+postmaster:
+  alias.present:
+    - target: root
+
+# pillar[[admin_username]] :
+#  alias.present:
+#    - target: root
```

Comment:
Why is this commented out? Can it be deleted?

Reply:
I have deleted this